src/python/grpcio_health_checking/commands.py

```diff
@@ -33,11 +33,17 @@
 import glob
 import os
 import os.path
+import shutil
 import subprocess
 import sys
 
 import setuptools
 from setuptools.command import build_py
+from setuptools.command import sdist
+
+ROOT_DIR = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
+HEALTH_PROTO = os.path.abspath(
+    os.path.join(ROOT_DIR, '../../../proto/grpc/health/v1/health.proto'))
```

Comment:
<sub>GitHub is eating my comments, it seems...</sub>
This ends up with `HEALTH_PROTO = $REPO_ROOT/proto/grpc/health/v1/health.proto`; one too many up-dirs.